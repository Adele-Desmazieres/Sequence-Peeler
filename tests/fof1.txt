../Tests/t3.fasta
../Tests/t4.fasta