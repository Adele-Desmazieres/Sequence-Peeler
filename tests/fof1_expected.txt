t3.fasta
t4.fasta